library/src/main/java/org/apache/apex/malhar/lib/wal/FSWindowDataManager.java

```diff
@@ -149,6 +149,8 @@
 
   private transient FileContext fileContext;
 
+  private SerializationBuffer serializationBuffer = new SerializationBuffer(new WindowedBlockStream());
```

Comment:
BlockStream is ok, SerializationBuffer constructor only take parameter with type WindowedBlockStream